internal/messaging/mint_v2.go

```diff
@@ -88,7 +87,13 @@ func (h *evmResponseHandler) prepareMintResponseV2(
 
 	h.logger.Infof("NFT minted with txID: %s\n", txID)
 
-	h.onBookingTokenMint(tokenID, response.MintId, response.BuyableUntil.AsTime())
+	tokenBoughtTimeout := time.Unix(response.BuyableUntil.Seconds, 0)
```

Comment:
Duplicated code in mintv1 + v2. This can be moved into a separate function called from every mint implementation.